iqm5q.py

```diff
@@ -24,7 +25,7 @@
 N_QUBITS = 5
 
 
-def create(runcard_path=RUNCARD):
+def create(runcard_path=RUNCARD, with_kernels: bool = True):
```

Comment:
Is there a case in which you want to avoid loading?

Reply:
Maybe you may want to still discriminate with threshold and angle. I am not aware of any case you would want to avoid them, theoretically they should provide better performance, I don't know if they actually do in our lab conditions.

Reply:
I'd be slightly bolder, and assume that if kernels are available, you should use them (dropping this further argument).
So, the choice of using kernels will depend on the platform, but not on the platform usage.

This will keep this interface simple :)

Reply:
I agree with @AleCandido. Anyway if they are not available we can replace them it an array of 1s, right @Jacfomg?

Reply:
This array of 1s was already being generated inside the zurich driver both for integration and its use with threshold and angle for discrimination. i think there is no need to use the `Kernels` for this and dump the arrays of 1s, right ?